qiita_pet/templates/sitebase.html

```diff
@@ -57,6 +57,7 @@
         'Chrome': {'version': 49, 'download': 'https://www.google.com/chrome/browser/'},
         'Firefox': {'version': 48, 'download': 'https://support.mozilla.org/en-US/kb/update-firefox-latest-version'},
         'Safari': {'version': 12, 'download': 'http://support.apple.com/en-us/HT6104'},
+        'Chrome Mobile': {'version': 69, 'download': 'https://www.google.com/chrome/browser/'},
```

Comment:
Both Chrome-related links are the same. Just to confirm, it's the site that intelligently figures out what to offer? 'Chrome Mobile' is just a needed synonym for 'Chrome' for Browser tags, etc? It also looks like one is 69 and one is 49.

Reply:
afaik, yup, and the numbers are right ...

Reply:
ok. it sounds like everything is as it should be.